src/openrct2-ui/windows/Staff.cpp

```diff
@@ -1472,12 +1472,13 @@ void window_staff_options_mousedown(rct_window* w, rct_widgetindex widgetIndex,
     int32_t numCostumes = staff_get_available_entertainer_costume_list(_availableCostumes);
     for (int32_t i = 0; i < numCostumes; i++)
     {
-        uint8_t costume = _availableCostumes[i];
-        if (peep->SpriteType == PEEP_SPRITE_TYPE_ENTERTAINER_PANDA + costume)
+        EntertainerCostume costume = _availableCostumes[i];
+        uint8_t costumeValue = static_cast<uint8_t>(costume);
+        if (peep->SpriteType == PEEP_SPRITE_TYPE_ENTERTAINER_PANDA + costumeValue)
```

Comment:
```suggestion
        if (peep->SpriteType == EntertainerCostumeToSprite(costume)
```
This can simplified as above, can't it? 
Then costumeValue can be moved to the inside the if (or just be dropped in favour of a direct cast in its single usage

Reply:
That makes sense; we can remove the costumeValue line. Done. I don't see any other extraneous uses of PEEP_SPRITE_TYPE_ENTERTAINER_PANDA.